blocks/newslist/newslist.js

```diff
@@ -327,6 +327,14 @@ async function updateYearsDropdown(block, articles) {
   addEventListenerToFilterYear(document.getElementById('filter-year'), window.location.pathname);
 }
 
+function ValidateSearchFormData(keyword) {
+  if (typeof keyword === 'undefined' || keyword === null) {
+    return '';
+  }
+  const sanitezedKeyword = keyword.replace(/[`%$^*()_+=[\]{}\\|<>/~]/g, '');
```

Comment:
why not just add all non-word/number characters to the regex and convert it to a space? like this ->
`const sanitezedKeyword = keyword.replace(/\W+/g, ' ');`

Reply:
we i understand your point. however, we cannot simply replace all special characters with whitespace, as there are words that include special characters. Additionally, we provide support for other GEOs that have native special characters in their words. see some examples below.

**Keyword:** Geschäftsmodell in Europa zu etablieren 
**all special character replaced with whitespace** --> Results for "DTC Gesch ftsmodell in Europa zu etablieren
**Keyword:** riscaldamento globale a +1,5 °C 
**all special character replaced with whitespace** --> Results for "riscaldamento globale a 1 5 C"
**Keyword:** promoverá a próxima era em criatividade, 
**all special character replaced with whitespace** -->  Results for "promover a pr xima era em criatividade"
**Keyword:** $94 million U.S. 
**all special character replaced with whitespace** --> Results for "94 million U S"